test/test_va_api_createbuffer.cpp

```diff
@@ -166,9 +168,13 @@ TEST_P(VAAPICreateBuffer, CreateBufferWithOutData)
                 currentTestInput.inputConfig.second);
 
             // vaCreateContext input requires resolution, since this test
-            // doesn't create surfaces, passing 1x1 resolution should provide
+            // doesn't create surfaces, passing min resolution should provide
             // the desired result.
-            doCreateContext(Resolution(1, 1));
+            doGetMinSurfaceResolution(
```

Comment:
`doGetMinSurfaceResolution` was removed in recent commit.  Please rebase and use new` getMinMaxSurfaceResolution` instead.